include/odp/api/spec/queue.h

```diff
@@ -88,6 +88,20 @@ int odp_queue_create_multi(const char *name[], const odp_queue_param_t param[],
  */
 int odp_queue_destroy(odp_queue_t queue);
 
+/**
+ * Destroy multiple queues
+ *
+ * Otherwise like odp_queue_destroy(), but destroys multiple queues with a
+ * single call.
+ *
+ * @param queue    Array of queue handles
+ * @param num      Number of queues to destroy
+ *
+ * @retval Number of queues actually destroyed (0 ... num)
```

Comment:
In destroy direction APIs, 0 may not be helpful return value. If first handle is invalid, function should return -1. Also function should be always able to free good handles.

Reply:
Changed in V4.